integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -208,6 +208,31 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
         System.out.println("Launching task " + taskId.getValue() + " using offer "
             + offer.getId().getValue());
 
+        Protos.Resource.Builder portsBuilder = Protos.Resource.newBuilder()
+            .setName(Constants.MESOS_RESOURCE_PORTS)
+            .setType(Protos.Value.Type.RANGES);
+
+        if (executorBuilder.getSource().equals("worker")) {
```

Comment:
make "worker" and "master" named constants and use the named constant here and on lines 166 and 167